src/Compilers/Core/Portable/SourceGeneration/GeneratorAdaptor.cs

```diff
@@ -52,16 +47,9 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                     // So we probably need an internal version that doesn't wrap it? Maybe we can just construct the nodes manually.
```

Comment:
Consider naming the lambda parameter `generatorExecutionContext` instead of `context`. There are a lot of different context items in this method and the naming helps make the code a bit more readable.